src/seq/fasta.rl

```diff
@@ -0,0 +1,251 @@
+
+
+# FASTA sequence types
+
+@doc """
+Metadata for FASTA sequence records containing just a `description` field.
+""" ->
+type FASTAMetadata
+    description::String
```

Comment:
My impression has been that descriptions are fairly common. E.g. Ensembl, miRBase, NCBI, SGD, FlyBase all use them. Do you think there would be a practical benefit to using Nullable to indicate no description, as opposed to just an empty string?